app/assets/javascripts/discourse/app/components/user-status-picker.js

```diff
@@ -0,0 +1,50 @@
+import Component from "@ember/component";
+import { action, computed } from "@ember/object";
+import { scheduleOnce } from "@ember/runloop";
+import { emojiUnescape } from "discourse/lib/text";
+import { escapeExpression } from "discourse/lib/utilities";
+
+export default class UserStatusPicker extends Component {
+  isFocused = false;
+  emojiPickerIsActive = false;
+  emoji = null;
+  description = null;
+
+  @computed("emoji")
+  get emojiHtml() {
+    const emoji = escapeExpression(`:${this.emoji}:`);
```

Comment:
what if emoji becomes null?

Reply:
I understand there's the default, but this is a function call, what if something else makes it null without calling the default function?

Reply:
I'm not strongly against handling possible null value here, but I'm not sure that we should be that protective. 

We do not support user statuses without emojis, an attempt to send a status without emoji to the server will lead to a server error and now `emojiHtml()` gets called only when emoji is already set:
```hbs
{{#if @emoji}}
   {{html-safe this.emojiHtml}}
{{else}}
```

Yes, something else can set emoji to null, but the answer to the question _What will happen then?_ is _Tests will fail_. And after seeing that failure, we'll need to prevent that thing from setting emoji to null.

We can add some protection and start returning an empty string from `emojiHtml()`. The modal will look like this then:

<img width="411" alt="Screenshot 2022-06-09 at 21 55 51" src="https://user-images.githubusercontent.com/1274517/172916265-594eb6d2-4b76-48fd-848d-81d20464a16e.png">

But in that case, tests will still fail and show us that we have a problem, so probably it doesn't worth adding additional code.

What do you think?